src/main/java/github/weichware10/toolbox/zoommaps/ZoomCalculator.java

```diff
@@ -1,8 +1,42 @@
 package github.weichware10.toolbox.zoommaps;
 
+import github.weichware10.util.Data;
+
 /**
- * TODO: Sarah.
+ * Bekommt Rohdaten des Systems (Mausklick), berechnet neue Position des Bildes
+ * und meldet Speichererfolg.
  */
 public class ZoomCalculator {
+    private float speed;
+    private float[] position;
+    private Data data;
+
+    /**
+     * verarbeitet Input von Benutzer/System.
+     *
+     * @param rawInput - Screenkoordinaten vom Mausklick (float[2])
+     */
+    public void processInput(int[] rawInput) {
```

Comment:
Hier können wir ein Platzhalter Test einbauen, und den mit @Test @Ignore annotieren, damit der Test geskipped wird
Und in dem Javadoc von dem Test schreiben wir, was der Test überprüfen soll